2025-XX-XX-qubes-os-4-3-0-rc3-available-for-testing.md

```diff
@@ -0,0 +1,51 @@
+- Default Whonix templates upgraded to Whonix 18 (older releases not supported).
+- Preloaded disposables ([#1512](https://github.com/QubesOS/qubes-issues/issues/1512))
+- Device "self-identity oriented" assignment (a.k.a. New Devices API) ([#9325](https://github.com/QubesOS/qubes-issues/issues/9325))
+- Qubes Windows Tools reintroduced with improved features ([#1861](https://github.com/QubesOS/qubes-issues/issues/1861)).
+
+These are just a few highlights from the many changes included in this release. For a more comprehensive list of changes, see the [Qubes OS 4.3 release notes](https://doc.qubes-os.org/en/latest/developer/releases/4_3/release-notes.html).
+
+## When is the stable release?
+
+That depends on the number of bugs discovered in this RC and their severity. As explained in our [release schedule](https://doc.qubes-os.org/en/latest/developer/releases/version-scheme.html#release-schedule) documentation, our usual process after issuing a new RC is to collect bug reports, triage the bugs, and fix them. If warranted, we then issue a new RC that includes the fixes and repeat the process. We continue this iterative procedure until we're left with an RC that's good enough to be declared the stable release. No one can predict, at the outset, how many iterations will be required (and hence how many RCs will be needed before a stable release), but we tend to get a clearer picture of this as testing progresses.
+
+At this time, we expect that there will likely be a fourth release candidate, which will probably be the final one.
+
+## How to test Qubes 4.3.0-rc3
+
+Thanks to those who tested earlier 4.3 RCs and [reported bugs they encountered](https://doc.qubes-os.org/en/latest/introduction/issue-tracking.html), 4.3.0-rc3 now includes fixes for several bugs that were present in those prior RCs!
+
+If you'd like to help us [test](https://doc.qubes-os.org/en/latest/user/downloading-installing-upgrading/testing.html) this RC, you can [upgrade to Qubes 4.3.0-rc3](https://doc.qubes-os.org/en/latest/user/downloading-installing-upgrading/upgrade/4_3.html) with either a clean installation or an in-place upgrade from Qubes 4.2. (Note for in-place upgrade testers: `qubes-dist-upgrade` now requires `--releasever=4.3` and may require `--enable-current-testing` for testing releases like this RC.) As always, we strongly recommend [making a full backup](https://doc.qubes-os.org/en/latest/user/how-to-guides/how-to-back-up-restore-and-migrate.html) beforehand and [updating Qubes OS](https://doc.qubes-os.org/en/latest/user/how-to-guides/how-to-update.html) immediately afterward in order to apply all available bug fixes.
+
+Existing 4.3 RC users can [update normally](https://doc.qubes-os.org/en/latest/user/how-to-guides/how-to-update.html) with [`current-testing` enabled](https://doc.qubes-os.org/en/latest/user/downloading-installing-upgrading/testing.html#updates) in order to make their systems effectively equivalent to 4.3.0-rc3.
```

Comment:
@adrelanos, @ArrayBolt3: Should we add (or link to) some instructions here for existing RC users to upgrade from Whonix 17 to 18? If so, what (or where) are those instructions?

Reply:
> https://www.whonix.org/wiki/Release_Upgrade_16_to_17

I see that https://www.whonix.org/wiki/Release_Upgrade_17_to_18 also exists, and I assume that's the one you probably meant to link, so I've add that one to the announcement.